getting-started/templates/systemlink-admin-values.yaml

```diff
@@ -16,3 +16,9 @@ global:
   # <ATTENTION> - Set to false if you do not want to manage secrets as part of the Helm installation.
   ##
   deploySecrets: true
+
+## Uncomment and adjust installation of Argo Workflows CRDs. Disable the installation of Argo Workflows CRDs
```

Comment:
Just trying to make sure I understand the behavior here, because I think I want to adjust the wording a little. 

- By default, we install Argo CRDs when the SystemLink-admin Helm chart is installed.
- If a user already has Argo installed in their cluster they will not want the admin helm chart to attempt to install it again.
- To resolve this, the user uncomments this section and sets `install` to `false`

Reply:
It's important to differentiate between Argo Workflows CRDS and Argo Workflows deployments. CRDS are globally to the cluster, where the deployments are scoped to a namespace.

In order for Argo Workflows to work, CRDS needs to be deployed first.

We had one customer that had Argo Workflows deployed before SystemLink, and when he tried to deploy SystemLink admin chart, it failed because of the CRDS conflict. In that case the only workaround was to disable argo crds deployment. Still, it's a risk the customer needs to assume because it's not the CRDS we shipped and tested with. Most likely executions will work, but if they not, customer will have to fix the CRDS.

It's also important to mention that we still deploy our argo workflows deployment, but it's scoped to just notebook executions.

Reply:
I cleaned up the wording of the comment and also changed `true` to `false` since this, from my understanding, is why a user would uncomment this section in the first place.